Assets/Scripts/Managers/PlayerManager.cs

```diff
@@ -5,7 +5,11 @@ namespace Managers
     public class PlayerManager : Singleton<PlayerManager>
     {
         [SerializeField] private GameObject player;
-        
+
         public GameObject GetPlayer() => player;
+
+        public bool PlayerInReach(GameObject otherObject, float distanceToPlayer) =>
```

Comment:
Maybe rename this to something like `reachDistance` or `maxRange` to make it clearer what's actually meant? And/Or add documentation :P